### PR TITLE
Add Unit tests 

### PR DESCRIPTION
**Pull request by mckadesorensen**

### CODE REVIEW

File: tests/test_granule_to_sns.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import boto3
+import pytest
+
+from granule_to_sns import generate_message, granule_to_sns
+
+
+@pytest.fixture
+def get_event():
```
Comment (reweeden) [inline tests/test_granule_to_sns.py]: Fixtures names should generally just be nouns. See the examples in the docs: https://docs.pytest.org/en/8.2.x/explanation/fixtures.html#about-fixtures

```suggestion
def event():
```

File: tests/test_granule_to_sns.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                        },
+                    ],
+                },
+            ],
+        },
+    }
+
+
+@pytest.fixture
+def get_message():
```
Comment (reweeden) [inline tests/test_granule_to_sns.py]:
```suggestion
def message():
```

File: tests/test_granule_to_sns.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            ],
+        },
+    }
+
+
+@pytest.fixture
+def sns_client(mocker):
+    sns_client = boto3.client("sns", region_name="us-west-2")
+    mocker.patch("boto3.client", return_value=sns_client)
+    return sns_client
```
Comment (reweeden) [inline tests/test_granule_to_sns.py:99]: Could consider using `moto`. Might make things simpler.

**Review by gjclark — APPROVED** [review]

File: tests/test_granule_to_sns.py (hunk 3 above)
Comment (gjclark) [reply tests/test_granule_to_sns.py:99]: I don't think this is egregious but I would say something like this is a little easier to read and consistent with how we do it elsewhere:
```python
@pytest.fixture
def sns():
    with mock_aws():
        yield boto3.client("sns", region_name="us-west-2")
```

**Review by reweeden — APPROVED** [review]

**Pull request merged**